### DISCUSSION TITLE
[Suggestion] Rotator direction indicator

### DISCUSSION THREAD

**Discussion by ZachZent** [tsunamayo/Starship-EVO#3968] [Suggestions]
When you place the rotator you are given the arrow to show the direction of rotation. However after placement, all sides look the exact same so there is no way besides trial and error to figure out the orientation of the rotator. Is there a possible way to add some sort of indicator in the model like a small mark on top or perhaps something similar to pipes where holding either another rotator or the link tool shows the orientation (world also work with other logic parts)?

**Comment by ProPeach** [comment]
Excuse the crappy job, but turning the double ended arrow in the placement ghost into a single headed arrow would be a great help on top of an arrow on the side of the actual rotator
![image](https://user-images.githubusercontent.com/27857711/117552640-950f3a80-b044-11eb-92d2-67d8379a0d19.png)

**Comment by ZachZent** [comment]
I was more meaning after the block is placed, but that would be helpful too

**Comment by ExodistSKY1** [comment]
Having the ability to choose the direction it rotates in that would be ideal

**Comment by TheShadowSage** [comment]
> Having the ability to choose the direction it rotates in that would be ideal

That would help a lot!

**Comment by tsunamayo** [comment]
@ZachZent I see, the problem is I wanted to give the option to offset that dead orientation. But I could simply allow moving below 0deg and beyond 360 deg. I will think about this a bit more.
